modules/events/infrastructure/drizzle/occasion-invitation-candidate-repository-drizzle.ts

```
@@ -1,4 +1,5 @@
 import { and, eq, ilike, isNull, lt, or } from "drizzle-orm";
+import { and, eq, ilike, isNotNull, isNull, lt, or, sql } from "drizzle-orm";
```

Biome (2.4.7)

[error] Shouldn't redeclare 'and'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)